### DISCUSSION TITLE
Help with Carpet on 1.17 Server

### DISCUSSION THREAD

**Discussion by TheOleJD** [gnembon/fabric-carpet#949] [Q&A]
As stated above, me and a friend have a Minecraft 1.17 server that has Fabric, Fabric API, and Carpet mod but I need help to get it working correctly. Some of the commands simply do not show up when doing '/carpet'. On the server, there are about 15-20%(if that) of the total commands that there would, normally, be there if I were to go into my single player world. Any suggestions as to why/what needs to  be done in order to get access to the rest of the commands?

**Comment by James103** [comment]
1. Did you also install carpet-extra? Some of the carpet rules that you may be referring to are exclusive to the carpet-extra mod.
2. Exactly which mods do you have installed on the client side as well?

**Reply by TheOleJD** [reply]
Well now that you mention that, I do not think we have carpet-extra installed. I would have thought that we installed it but we skipped over it in the rush to get started apparently. Thank you for your help!
